MOBILE_CLIENT/COMMON/_HIGH/INSECURE_VIRUSTOTAL_SCAN/recommendation.md

```diff
@@ -0,0 +1,10 @@
+If you discover that one of the assets associated with your application has a bad reputation or is flagged as malicious, it's important to take action to address the issue and ensure the security and reliability of your application.
+Actions you need to take will depend on the nature of the asset, your control over it, and the severity of the reported issue.
+
+Here are some recommendations on what to do in such a situation:
+
+ * Determine why the asset was flagged as malicious. It could be due to various factors, such as hosting malicious content, containing malware, spam, or being associated with a history of cyberattacks. Investigate the specifics of the problem.
+
+ * Consider isolating it from your primary systems. You may want to temporarily disable or quarantine the asset to prevent further potential issues while you investigate.
+
+ * If the asset is associated with a service you control, take steps to patch and secure it. This might involve updating software, implementing stronger security measures, and ensuring it is not used for malicious purposes.
```

Comment:
This feels more like the reputation KB entry.

Reply:
they are basically the same